tests/Microsoft.Bot.Builder.Dialogs.Declarative.Tests/ResourceTests.cs

```diff
@@ -9,22 +9,14 @@
 using Microsoft.Bot.Builder.Dialogs.Debugging;
 using Microsoft.Bot.Builder.Dialogs.Declarative.Debugging;
 using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
-using Microsoft.VisualStudio.TestPlatform.Utilities;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Newtonsoft.Json;
+using Xunit;
+using Xunit.Sdk;
 
 namespace Microsoft.Bot.Builder.Dialogs.Declarative.Tests
 {
-    [TestClass]
     public class ResourceTests
     {
-        private static JsonSerializerSettings jsonSerializerSettings =
-            new JsonSerializerSettings() { NullValueHandling = NullValueHandling.Ignore, Formatting = Formatting.Indented };
-
-        public TestContext TestContext { get; set; }
-
-        [ClassInitialize]
```

Comment:
I think we should use IClassFixture replacing ClassInitialize so it is executed once before running all the tests in this class.

Reply:
Feedback applied. Thanks!